docs/source/reference/training-guide.rst

```diff
@@ -97,6 +97,24 @@ Use high-performance networking
         See `GPUDirect-TCPX example <https://docs.skypilot.co/en/latest/examples/performance/gcp_gpu_direct_tcpx.html>`_ for more details.
 
 
+Using Ray with SkyPilot
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Ray workloads on SkyPilot:
+
+- **Running** ``ray start`` **is fine** - Ray defaults to port 6379, while SkyPilot uses port 6380 internally
+- **Never use** ``ray.init(address="auto")`` - It connects to SkyPilot's internal Ray cluster
+- **Start Ray head on rank 0, workers on other ranks** - See :ref:`distributed Ray example <dist-jobs>`
```

Comment:
We should refer to the `distributed Ray example` at the beginning of this section. And just move these important notes to that page, as these notes are a bit hard to understand if not coming with the concrete example.